osbuild/markup.py

```diff
@@ -0,0 +1,6 @@
+RESET = "\033[0m"
+
+BOLD = "\033[1m"
+
+RED = "\033[31m"
+GREEN = "\033[32m"
```

Comment:
I don't mind extracting them like this, but maybe as a followup we could group them better into a class like `vt`, that collects all vt-escape-sequences? Then we avoid importing each individual one everywhere and can just resort to `from markup import vt`?

Reply:
I was anticipating the same follow up with the addition of an isatty() call to nullify the formatting if we're not outputting to a tty.